### PR TITLE
Attempted to fetch from my Spotify's API

### PR DESCRIPTION
**Pull request by elcruzo**

### CODE REVIEW

File: Backend/prisma/schema.prisma [hunk 1]
```diff
@@ -14,9 +14,13 @@ datasource db {
...
-  isVerified      Boolean   @default(false)
+  id                    Int       @id @default(autoincrement())
+  username              String    @unique
+  email                 String    @unique
+  password              String
+  isVerified            Boolean   @default(false)
+  spotifyAccessToken    String?   // is null if user might has not authenticated with Spotify
+  spotifyRefreshToken   String?   // is null if user might has not authenticated with Spotify
+  topArtists            Json?     // stores top artists as JSON
+  topTracks            Json?     // stores top tracks as JSON
```
Comment (ahdahl) [inline Backend/prisma/schema.prisma:25]: thanks for adding comments here, this is helpful

File: Backend/routes/userRoutes.js [hunk 2]
```diff
@@ -79,4 +85,176 @@ router.get("/info", authenticateJWT, async (req, res) => {
...
+            grant_type: 'authorization_code',
+        }),
+        headers: {
+            'Authorization': 'Basic ' + Buffer.from(CLIENT_ID + ':' + CLIENT_SECRET).toString('base64'),
+            'Content-Type': 'application/x-www-form-urlencoded'
+        }
+    };
+
+    try {
+        console.log(authOptions)
```
Comment (ahdahl) [inline Backend/routes/userRoutes.js]: Do not commit console.log statements

**Review by ahdahl — APPROVED** [review]
Great job hooking up the spotify API, this is complex work! Please remove the console.log statements before merging into main

**Comment by ahdahl** [comment]
Also, can you please add some details to the test plan?

**Comment by elcruzo** [comment]
<div>
    <a href="https://www.loom.com/share/252f95b4ef634b8bb8e43547366aab4a">
      <p>Demo of User Authentication</p>
    </a>
    <a href="https://www.loom.com/share/252f95b4ef634b8bb8e43547366aab4a">
      
    </a>
  </div>

**Pull request merged**